### PR TITLE
Refactor apply_yaml action to use ObjectStore.CreateFromYAML

### PR DESCRIPTION
**Pull request by wwitzel3**
Signed-off-by: Wayne Witzel III <wayne@riotousliving.com>

**What this PR does / why we need it**:

Make it easier for other parts of Octant to take advantage of the CreateFromYAML logic that was added in #954

### CODE REVIEW

**Comment by wwitzel3** [comment]
related #1136

File: pkg/store/store.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ type Store interface {
 	Update(ctx context.Context, key Key, updater func(*unstructured.Unstructured) error) error
 	IsLoading(ctx context.Context, key Key) bool
 	Create(ctx context.Context, object *unstructured.Unstructured) error
+	CreateFromYAML(ctx context.Context, namespace, input string) ([]string, error)
```
Comment (scothis) [inline pkg/store/store.go:41]: "Create" is only half of what this call actually does.

```suggestion
	ApplyFromYAML(ctx context.Context, namespace, input string) ([]string, error)
```

or 

```suggestion
	CreateOrUpdateFromYAML(ctx context.Context, namespace, input string) ([]string, error)
```

**Review by scothis — APPROVED** [review]
Looks good. One small quibble with the method name

File: pkg/store/store.go (hunk 1 above)
Comment (wwitzel3) [reply pkg/store/store.go:41]: Yep, this is better. I'll change it.

**Review by scothis — APPROVED** [review]

**Pull request merged**